cartography/data/jobs/cleanup/azure_storage_account_cleanup.json

```diff
@@ -0,0 +1,95 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:AzureStorageAccount)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (:AzureStorageAccount)<-[r:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (n:AzureStorageQueueService)<-[:USES]-(:AzureStorageAccount)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
I understand this kind of MATCH was done before, but we should be sure it is even necessary, else it's misleading to future contributors.
`<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) `

Reply:
@ramonpetgrave64 This condition helps in multi-subscription setup. When a user has multiple subscriptions, this condition would make sure that resources from other subscription of the same user are not picked up while running query. with the subscription id condition, the scope is restricted to resources of one specific subscription vs the entire tenant. helps in a multi-tenant setup

Reply:
Alright. Thanks for the explanation.